guides/common/attributes-titles.adoc

```diff
@@ -10,7 +10,7 @@
 :InstallingServerDocTitle: Installing {ProjectServerTitle} {ProjectVersion} on {install-on-os}
 // Installing Disconnected - defined for Satellite only
 :InstallingServerDisconnectedDocTitle: Installing {ProjectServerTitle} in a Disconnected Network Environment
-:InstallingSmartProxyDocTitle: Installing a {SmartProxyServerTitle} {ProjectVersion} on {install-on-os}
+:InstallingSmartProxyDocTitle: Installing a {SmartProxy}{nbsp}Server {ProjectVersion} on {install-on-os}
```

Comment:
Why now use the same as line 57?

Reply:
I just replaced what it renders to currently ("Server" vs "server")